phoenix-core/src/main/java/org/apache/phoenix/schema/types/PDataType.java

```diff
@@ -519,6 +519,74 @@ protected Random initialValue() {
         }
     };
 
+    // Ordinal list. It makes easy to "reorder" cardinals. for instance, UUID and UUID_INDEXABLE
```

Comment:
Sorry, I don't quite understand the question.
When I say 'reorder' I mean that if we need to reorder other cardinals in the future (for example, let's imagine that we want to add another type 'ipnetwork'), since they are constants in the file, it would be easier. For example, by incorporating the ORDINAL_UUID and ORDINAL_UUID_INDEXABLE, this way is easy.